pactum/resources.py

```diff
@@ -8,21 +11,48 @@ def __init__(self, name=None):
 class Resource(BaseResource):
     def __init__(self, name=None, fields=None, behaviors=None):
         super().__init__(name)
+
         self._mapfields = {}
+        self.fields = []
+        self._load_fields(fields)
 
+        if behaviors is None:
+            behaviors = getattr(self, "behavior", [])
+        self.behaviors = behaviors
```

Comment:
```suggestion
        self.behaviors = behaviors if behaviors is not None else getattr(self, "behavior", [])
```
ou
```suggestion
        self.behaviors = behaviors or getattr(self, "behavior", [])
```

Nao ficaria mais "pythonico"?

Reply:
Acho que ficaria mais difícil de ler, não acha?

Reply:
Além disso, no segundo, tem um problema com parâmetros "falsy" em behavior: `Resource(behavior="")` vai acionar o `getattr()` porque não valida explicitamente a presença do `None`.

Reply:
então, mas é interessante tem qualquer outro tipo de valor "falsy" nesse cara? não parece ser o caso...

Reply:
sobre legibilidade, eu particularmente gosto de operadores ternários, mas como disse, particularmente.

Reply:
> sobre legibilidade, eu particularmente gosto de operadores ternários, mas como disse, particularmente.

eu não curto muito porque joga para segundo plano (mais à direita da tela) a lógica...

Reply:
talvez colocar essa implementacão de behaviors em outro PR valha a pena.
Nesse caso colocaria a implementacão já testada.

Reply:
Me parece que já tinha esse "comportamento" antes, só moveu de lugar...

Reply:
O problema era que o `.__init__()` já recebia behaviors como parâmetro (e os testes já passavam behaviors como parâmetro)... só repliquei a lógica antiga do fields para não ficar com código morto.

O patch para remover os vestígios de behaviors que já existiam no pactum seria maior do que esse.

Reply:
É, parece que essa semente já estava plantada antes, mas ainda não tá testada...
@osantana confirma que essa mudanca vai ser melhor resolvida no próximo PR que pra mim tudo bem.